main.go

```diff
@@ -33,8 +33,12 @@ const (
 	emailIDEnv        = "EMAIL_ID"
 	emailPasswordEnv  = "EMAIL_PASSOWORD"
 	searchIntervalEnv = "SEARCH_INTERVAL"
+	VaccineEnv        = "VACCINE"
```

Comment:
```suggestion
	vaccineEnv        = "VACCINE"
```
Let's avoid exporting if not required